From 82cdf88aa9a9940a3dcd6633786f218e402f7feb Mon Sep 17 00:00:00 2001
From: Sudip Mukherjee <sudipm.mukherjee@gmail.com>
Date: Wed, 22 Apr 2020 20:08:15 +0100
Subject: [PATCH 13/13] soundwire: intel: Remove unused function

The function sdw_intel_init() is not used anywhere, remove it for now.

Signed-off-by: Sudip Mukherjee <sudipm.mukherjee@gmail.com>
Link: https://lore.kernel.org/lkml/20200422190815.5975-1-sudipm.mukherjee@gmail.com/
Signed-off-by: Nathan Chancellor <natechancellor@gmail.com>
---
 drivers/soundwire/intel_init.c | 23 -----------------------
 1 file changed, 23 deletions(-)

diff --git a/drivers/soundwire/intel_init.c b/drivers/soundwire/intel_init.c
index ad7053463889..2f37dad06321 100644
--- a/drivers/soundwire/intel_init.c
+++ b/drivers/soundwire/intel_init.c
@@ -182,29 +182,6 @@ static acpi_status sdw_intel_acpi_cb(acpi_handle handle, u32 level,
 	return AE_CTRL_TERMINATE;
 }
 
-/**
- * sdw_intel_init() - SoundWire Intel init routine
- * @parent_handle: ACPI parent handle
- * @res: resource data
- *
- * This scans the namespace and creates SoundWire link controller devices
- * based on the info queried.
- */
-static void *sdw_intel_init(acpi_handle *parent_handle,
-			    struct sdw_intel_res *res)
-{
-	acpi_status status;
-
-	status = acpi_walk_namespace(ACPI_TYPE_DEVICE,
-				     parent_handle, 1,
-				     sdw_intel_acpi_cb,
-				     NULL, res, NULL);
-	if (ACPI_FAILURE(status))
-		return NULL;
-
-	return sdw_intel_add_controller(res);
-}
-
 /**
  * sdw_intel_exit() - SoundWire Intel exit
  * @arg: callback context
-- 
2.26.2

